### PR TITLE
Rollbacked to nested JSON form with "base_req" keyword

### PR DESCRIPTION
**Pull request by psy2848048**
## Update
* `base_req` JSON key category rebirthed

## Good to review
1. Install [Postman](https://www.postman.com/)
1. Launch `nodef start` & `clif rest-server`
1. Test the endpoints below

### `POST /hdac/transfer`
```json
{
	"base_req": {
		"chain_id": "testnet",
		"gas": "20000000",
		"memo": "",
		"from": "princesselsa"
	},
	"recipient_address_or_nickname":"friday1y2dx0evs5k6hxuhfrfdmm7wcwsrqr073htghpv",
	"amount":"1000000",
	"fee": "100000000"
}
```

### `POST /hdac/bond     /    /hdac/unbond`
```json
{
	"base_req": {
		"chain_id": "testnet",
		"gas": "20000000",
		"memo": "",
		"from": "princesselsa"
	},
	"fee": "100000000",
	"amount":"1000000"
}
```

### `POST PUT /hdac/validators`
```json
{
	"base_req": {
		"chain_id": "testnet",
		"gas": "20000000",
		"memo": "",
		"from": "princesselsa"
	},
 "cons_pub_key":"fridayvalconspub16jrl8jvqq9cj7569gec8sd6dv4e4w46g25uk7vm52fpk6nfn23p5zv60xekxuse3df39sjz4g9d92nfc0f3rs3ec2fpxxum9wej52428w968xwzww9erw53ng5u9zu23fdvk226z0fnx5mn2f4qk542ygf9xjvptve6y7a6fxe8kjm6s2e3kja2twezyvmt5245ystm4ga94y4zjwfg523c477wna",
	"description":{
		"website": "https://hdac.io",
		"moniker": "testnode1"
	}
}
```

### CODE REVIEW

**Comment by yoosah** [comment]
title has a typo.. 'neted' :)

File: x/executionlayer/client/rest/msgCreator.go [hunk 1]
```diff
@@ -13,14 +14,12 @@ import (
...
-	TokenContractAddress       string `json:"token_contract_address"`
-	SenderAddressOrNickname    string `json:"sender_address_or_nickname"`
-	RecipientAddressOrNickname string `json:"recipient_address_or_nickname"`
-	Amount                     uint64 `json:"amount"`
-	Fee                        uint64 `json:"fee"`
-	GasPrice                   uint64 `json:"gas_price"`
-	Memo                       string `json:"memo"`
+	BaseReq                    rest.BaseReq `json:"base_req"`
+	TokenContractAddress       string       `json:"token_contract_address"`
+	SenderAddressOrNickname    string       `json:"sender_address_or_nickname"`
```
Comment (JoowonYun) [inline x/executionlayer/client/rest/msgCreator.go]: What do you think about the from address also go into baseReq?
I think from is an essential value when creating tx.

Comment (psy2848048) [reply x/executionlayer/client/rest/msgCreator.go]: https://github.com/hdac-io/friday/pull/70/commits/619b07f13770c808b45f2a8c12fb0efa8eb2fcff

Comment (JoowonYun) [reply x/executionlayer/client/rest/msgCreator.go]: How about add to token_address & fee in baseReq?

**Pull request merged**